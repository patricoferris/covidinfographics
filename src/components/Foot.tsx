import React from 'react'
import { Typography, Grid } from '@material-ui/core'

const Foot: React.SFC = () => {
  const link = {
    textDecoration: 'none',
    color: '#ffffff',
  }

  return (
    <Grid
      style={{
        backgroundColor: '#20232a',
        color: '#ffffff',
      }}
      container
      spacing={3}
    >
      <Grid item sm={false} md={2} />
      <Grid item sm={12} md={4}>
        <Typography variant="h6">Platforms</Typography>
        <ul>
          <li>
            <a style={link} href="https://www.facebook.com/COVID2019Infographics">
              Facebook
            </a>
          </li>
          <li>
            <a style={link} href="https://twitter.com/19Infographics">
              Twitter
            </a>
          </li>
          <li>
            <a style={link} href="https://www.instagram.com/covid19_infographics/">
              Instagram
            </a>
          </li>
        </ul>
      </Grid>
      <Grid item sm={12} md={4}>
        <Typography variant="h6">Sources</Typography>
        <ul>
          <li>
            <a style={link} href="https://www.nice.org.uk/covid-19">
              NICE
            </a>
          </li>
          <li>
            <a style={link} href="https://www.nhs.uk/conditions/coronavirus-covid-19/">
              NHS
            </a>
          </li>
          <li>
            <a style={link} href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019">
              WHO
            </a>
          </li>
        </ul>
      </Grid>
      <Grid item sm={false} md={2} />
      <Grid item sm={false} md={2} />
      <Grid item sm={12} md={4}>
        &copy; 2020 Copyright
      </Grid>
      <Grid item sm={12} md={4}>
        <a style={link} href="https://github.com/baizel/covidinfographics">
          Source Code
        </a>
      </Grid>
    </Grid>
  )
}

export default Foot
