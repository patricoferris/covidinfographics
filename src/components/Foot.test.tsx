import React from 'react'
import { shallow } from 'enzyme'
import Foot from './Foot'

test('Render footer correctly', () => {
  const wrapper = shallow(<Foot />)
  expect(wrapper).toMatchSnapshot()
})
