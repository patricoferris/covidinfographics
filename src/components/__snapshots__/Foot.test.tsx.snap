// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render footer correctly 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
  spacing={3}
  style={
    Object {
      "backgroundColor": "#20232a",
      "color": "#ffffff",
    }
  }
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={2}
    sm={1}
    xs={1}
  />
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={4}
    sm={10}
    xs={10}
  >
    <WithStyles(ForwardRef(Typography))
      variant="h6"
    >
      Platforms
    </WithStyles(ForwardRef(Typography))>
    <ul>
      <li>
        <a
          href="https://www.facebook.com/COVID2019Infographics"
          style={
            Object {
              "color": "#ffffff",
              "textDecoration": "none",
            }
          }
        >
          Facebook
        </a>
      </li>
      <li>
        <a
          href="https://twitter.com/19Infographics"
          style={
            Object {
              "color": "#ffffff",
              "textDecoration": "none",
            }
          }
        >
          Twitter
        </a>
      </li>
      <li>
        <a
          href="https://www.instagram.com/covid19_infographics/"
          style={
            Object {
              "color": "#ffffff",
              "textDecoration": "none",
            }
          }
        >
          Instagram
        </a>
      </li>
    </ul>
  </WithStyles(ForwardRef(Grid))>
  <Hidden
    mdUp={true}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={2}
      sm={1}
      xs={1}
    />
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={2}
      sm={1}
      xs={1}
    />
  </Hidden>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={4}
    sm={10}
    xs={10}
  >
    <WithStyles(ForwardRef(Typography))
      variant="h6"
    >
      Sources
    </WithStyles(ForwardRef(Typography))>
    <ul>
      <li>
        <a
          href="https://www.nice.org.uk/covid-19"
          style={
            Object {
              "color": "#ffffff",
              "textDecoration": "none",
            }
          }
        >
          NICE
        </a>
      </li>
      <li>
        <a
          href="https://www.nhs.uk/conditions/coronavirus-covid-19/"
          style={
            Object {
              "color": "#ffffff",
              "textDecoration": "none",
            }
          }
        >
          NHS
        </a>
      </li>
      <li>
        <a
          href="https://www.who.int/emergencies/diseases/novel-coronavirus-2019"
          style={
            Object {
              "color": "#ffffff",
              "textDecoration": "none",
            }
          }
        >
          WHO
        </a>
      </li>
    </ul>
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={2}
    sm={1}
    xs={1}
  />
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={2}
    sm={1}
    xs={2}
  />
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={4}
    sm={4}
    xs={4}
  >
    Â© 2020 Copyright
  </WithStyles(ForwardRef(Grid))>
  <WithStyles(ForwardRef(Grid))
    item={true}
    md={4}
    sm={4}
    xs={4}
  >
    <a
      href="https://github.com/baizel/covidinfographics"
      style={
        Object {
          "color": "#ffffff",
          "textDecoration": "none",
        }
      }
    >
      Source Code
    </a>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
